tests/client/amo/reducers/testFeatured.js

```diff
@@ -0,0 +1,66 @@
+import * as actions from 'amo/actions/featured';
+import featured, { initialState } from 'amo/reducers/featured';
+
+describe('featured reducer', () => {
+  it('defaults to loading', () => {
```

Comment:
But fortunately I only need to change the text, as it shouldn't default to loading 😄